packages/ui-voip/src/lib/VoipClient.ts

```diff
@@ -936,6 +943,33 @@ class VoipClient extends Emitter<VoipEvents> {
 		this.networkEmitter.emit('localnetworkoffline');
 		this.emit('stateChanged');
 	};
+
+	private getContactHostName(): string | undefined {
+		try {
+			const url = new URL(this.config.siteUrl);
+			return url.hostname;
+		} catch {
+			//
+		}
+	}
```

Comment:
Better to return undefined explicitly, however it's fine since this is private...